docs/data/data-grid/ai-assistant/ai-assistant.md

```diff
@@ -149,6 +149,51 @@ type Result<T> = { ok: false; message: string } | { ok: true; data: T };
 
 Your resolver should return `Promise<PromptResponse>`.
 
+## FAQs
+
+<details>
+<summary>With MUI service, where are the AI Assistant queries processed?</summary>
+<div style="margin-left: 0.5em">
```

Comment:
No custom CSS in the page, this should be managed at the docs level.

```suggestion
<div>
```

Reply:
We could inject a custom Material UI Accordion as an embedded demo, I imagine this would do it too.